components/Article/Page.js

```diff
@@ -622,6 +626,18 @@ class ArticlePage extends Component {
           showSubscribe: false
         })
       : undefined
+    const actionBarNavNew = actionBar
+      ? React.cloneElement(actionBar, {
+          animate: false,
+          estimatedReadingMinutes: undefined,
+          estimatedConsumptionMinutes: undefined,
+          onPdfClick: undefined,
+          pdfUrl: undefined,
+          showSubscribe: false,
+          fontSize: false,
+          wrapped: true
+        })
+      : undefined
```

Comment:
Instead of having `actionBarNavNew` and `actionBarNav` const it would be better to just have one and configure it differently depending on the `isTester` flag:

```suggestion
    const actionBarNav = actionBar
      ? React.cloneElement(actionBar, isTester ? {
          animate: false,
          estimatedReadingMinutes: undefined,
          estimatedConsumptionMinutes: undefined,
          onPdfClick: undefined,
          pdfUrl: undefined,
          showSubscribe: false,
          fontSize: false,
          wrapped: true
        } : {
          animate: false,
          estimatedReadingMinutes: undefined,
          estimatedConsumptionMinutes: undefined,
          onPdfClick: undefined,
          pdfUrl: undefined,
          showSubscribe: false
        })
      : undefined
```

Currently you accidentally use `actionBarNavNew` even with the old nav.